templates/security/cis.html

```diff
@@ -13,7 +13,8 @@
       <h1>CIS Benchmark on Ubuntu</h1>
       <h4>Comply with the most widely accepted baseline</h4>
       <p>The CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides certified tooling for audit and automated compliance with the CIS benchmarks with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
-      <a href="/security/certifications#get-in-touch" class="p-button--positive js-invoke-modal">Access the tooling for audit and automated CIS compliance</a>
+      <a href="/security/certifications#get-in-touch" class="p-button--positive js-invoke-modal">Contact us</a>
```

Comment:
Can you remove the js-invoke-modal class, as this will trigger the previous contact form. Thanks

Reply:
Triggering the contact form is the intention here. Do I miss something?